jwst/flatfield/tests/test_flatfield.py

```diff
@@ -76,35 +76,25 @@ def test_flatfield_step_interface(instrument, exptype):
     assert result.var_flat.shape == shape
     assert result.meta.cal_step.flat_field == "COMPLETE"
 
-
-def exptypes():
```

Comment:
why is so much random stuff removed from this test, like attribute setting for `data.meta` and this entire function?